test/entities/VideoEntityTest.java

```diff
@@ -0,0 +1,77 @@
+package entities;
+
+import com.google.api.client.util.DateTime;
+import org.junit.jupiter.api.Test;
+import org.junit.jupiter.api.BeforeEach;
+
+import static org.junit.jupiter.api.Assertions.*;
+
+class VideoEntityTest {
+
+    private Video video;
+
+    private Video longVideo;
+
+    @BeforeEach
+    void setup() {
+        DateTime dateTime1 = new DateTime("2020-02-19T18:48:38.000Z");
+        video = new Video("H-v6DfxnjF8",
+                "FIE Fencing Channel",
+                "Grand Prix Turin 2020 Women's Foil Final THIBUS FRA vs KIEFER USA",
+                "",
+                dateTime1,
+                12996,
+                169,
+                9);
+
+        DateTime dateTime2 = new DateTime("2020-02-19T18:48:38.000Z");
+        longVideo = new Video("QbwgQSwMSGM",
```

Comment:
Sorry if I'm not following here, but is this longVideo expected to be used?

Reply:
Oh my bad, it should be used in the checkDescriptionLength() method - nice catch!